max_iterations: 100000
max_generations: 100000
sample_size: 3
sample_type: "halton" # 'halton', 'random', or 'input'